packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -201,7 +206,7 @@ class Frame extends React.Component {
       type: 'text',
       className: cx('search-input', { 'is-hidden': isHidden }),
       required: required && !display.length ? true : undefined,
-      'aria-required': required && !display.length ? 'required' : undefined,
+      'aria-required': (required && !display.length),
```

Comment:
We may want to double check on this.

For screen readers these are not always identical. Some attributes shouldn't be present at all if they are false.

```
<input />
<input aria-require="false" />
```

This is also switching from this to this:

```
<input aria-require="required" />
<input aria-require="true" />
```

Reply:
cc: @bjankord @dev-cprice

Reply:
I made the change here because when I ran wdio tests I got `Invalid ARIA attribute: aria-required="required"` and we do similar settings in the other variants like `'aria-required': required,` which results in getting the `<input aria-required="false" />`

Reply:
Yeah, this is probably just a typo on my part on the big refactor. Should be using the `required` prop var instead of `'required'` (the string). `aria-required` should be a boolean value, defaulting to `"false"`